CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [2.0.0] - not released yet
 
+### Bug fixes
+
+- Now a thread/process is not delete when the task submitted take more time than maxInactiveTime configured (issue #70)
```

Comment:
`[...] not deleted [...]`
You can use google translator if you need help to produce better sentences

Reply:
Ahah come on, it's a typo

Reply:
![image](https://media.giphy.com/media/cOQSc9wAHifk1LlQBM/giphy.gif)

Reply:
@pioardi I'm sorry, I'm a human linter